plugins/processors/starlark/starlark.go

```diff
@@ -48,15 +48,51 @@ type State struct {
 	content map[string]interface{}
 }
 
+// The struct that defines a Tuple
+type tuple struct {
+	values []interface{}
+}
+
 // Load from the shared state, the value corresponding to the given key
 func (s *State) Load(key string) starlark.Value {
 	var value = s.content[key]
 	if value == nil {
 		return starlark.None
 	}
-	switch v := value.(type) {
+	return toValue(value)
+}
+
+// Convert the given value extrated from the shared state into a starlark value
+func toValue(original interface{}) starlark.Value {
+	switch v := original.(type) {
 	case telegraf.Metric:
 		return WrapMetric(v)
+	case []interface{}:
```

Comment:
seems like we might be missing a lot of types here. Might be better to use reflection to detect slices and maps